src/renderer/services/chain/fees/swap.ts

```diff
@@ -72,9 +63,9 @@ const swapFeeByChain$ = ({
 
     case ETHChain: {
       return FP.pipe(
-        type === 'source' && router
+        type === 'source' && routerAddress
```

Comment:
We might not need `type` anymore, but let's wait for feedback about swap fee handling ...